pkg/cloud/metadata/lingjun_test.go

```diff
@@ -0,0 +1,121 @@
+package metadata
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewLingJunMetadata(t *testing.T) {
+	t.Run("successful parsing", func(t *testing.T) {
+		// Prepare test data
+		testData := LingjunMetaData{
+			RegionId:          "cn-hangzhou",
+			ZoneId:            "cn-hangzhou-a",
+			NodeId:            "i-1234567890abcdef0",
+			InstanceType:      "ecs.g6.large",
+			EniMac:            "00:11:22:33:44:55",
+			AckNicName:        "eth1",
+			NimitzNetworkMode: "VPC",
+			NimitzInterface:   []string{"eth0", "eth1"},
+		}
+
+		data, err := json.Marshal(testData)
```

Comment:
I think we should copy a string from real LingJun node. Or we are just testing Go stdlib.